yarn.lock

```diff
@@ -1562,7 +1562,7 @@
     rehype-stringify "^8.0.0"
     remark-breaks "^2.0.2"
     remark-emoji "^2.1.0"
-    remark-parse "^8.0.3"
+    remark-parse-no-trim "^8.0.4"
```

Comment:
any context on this?  is upgrading the version on the standard package not an option?

Reply:
Unfortunately not an option: https://github.com/elastic/eui/issues/5543#issuecomment-1352060546

> Anything above `remark-parse@8.0.3` (what EUI is at) would be a **major breaking change** due to the remark team [completely changing their underlying parser in 9.x](https://github.com/remarkjs/remark/releases/tag/13.0.0). This breaks EUI's own tooltip and checkbox plugins, and additionally **will** break any other custom plugins that consumers are currently using (and AFAICT there is no clear or easy-to-follow migration guide written by the team).

> If the goal of this issue is **primarily** to stop the security warnings for `trim@0.0.1`: I would suggest an interim solution of forking `remark-parse` at 8.0.3 [...] In fact... it looks like GitHub has done exactly that already: https://github.com/docs/remark-parse-no-trim

I might suggest Kibana also consider using `remark-parse-no-trim` as well - just my 2c!